src/components/Dashboard/Dashboard.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
 import { Component } from 'react';
 import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
-
+import InputModal from '../resuableComponent/InputModal';
 
 class Dashboard extends Component {
   constructor(props) {
     super(props);
+    this.cellButtonL1 = this.cellButtonL1.bind(this);
+    this.cellButtonEval = this.cellButtonEval.bind(this);
+    this.showL1Results= false ;
+    this.showGkResults= false ;
+    
+  }
+
+  
+  cellButtonL1(cell, row,coloumindex, rowIndex) {
+    let theButton
+      theButton = <button style={{ backgroundColor: "none", border:"none" }}
+                       type="button"
+                       onClick={() => this.selectL1Eval(cell, row, rowIndex)}>
```

Comment:
I think Instead of repeating on each button this should hide/show pop up modal and we can use set State function